README.md

```diff
@@ -17,6 +17,39 @@ This application as of now fetches vaccine slots for only 1 configured district.
 Please follow [link 1] and [link 2] to create Telegram channel and get the API token from BotFather.
 Use the API key as mentioned above in application.yml file and Channel name from channel description (not the actual name)
 
+## Prerequisites
+- Install JDK 11
+- Set JAVA_HOME and JDK_HOME as system environment variables to installed location
+- Install Maven
+- Set MAVEN_HOME and M2_HOME to installed location
+- Add %Maven_HOME%\bin to path variable
+
+### Eclipse EE
+- Download latest Eclipse preferably (which supports JDK 11)
+- Lombok is refered in this project which requires Lombok to be installed on Eclipse.
+- To install lombok, it should be first downloaded to the repository with below command
```

Comment:
No need for individual library installation instructions , since lombok is included in pom , it will be downloaded and part of package just like spring and other library are

Reply:
Known issue with eclipse for this library and doesn't work without the workaround - https://stackoverflow.com/questions/11803948/lombok-is-not-generating-getter-and-setter